ext/TrixiNLsolveExt.jl

```diff
@@ -0,0 +1,124 @@
+# Package extension for adding NLsolve-based features to Trixi.jl
+module TrixiNLsolveExt
+
+# Required for finding coefficients in Butcher tableau in the third order of 
+# P-ERK scheme integrators
+if isdefined(Base, :get_extension)
+    using NLsolve: nlsolve
+else
+    # Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
+    using ..NLsolve: nlsolve
+end
+
+# We use a random initialization of the nonlinear solver.
+# To make the tests reproducible, we need to seed the RNG
+using StableRNGs: StableRNG, rand
+
+# Use functions that are to be extended and additional symbols that are not exported
+using Trixi: Trixi, solve_a_butcher_coeffs_unknown!, compute_c_coeffs, @muladd
```

Comment:
What if you are not importing `solve_a_butcher_coeffs_unknown!` here? Can you still "overload" in line 75?

Reply:
It works! But this won't be consistent with how we import functions and overwrite them in TrixiConvexECOSExt but let's see what Aqua.jl has to say about this.